src/ray/util/common_runtime/test/CMakeLists.txt

```diff
@@ -1,13 +1,21 @@
 SET(CMAKE_CXX_STANDARD 11)
 
+include(../../../../../thirdparty/scripts/thirdparty.cmake)
```

Comment:
We may put this inside the root cmakefile of common runtime, so that later on when the path of the thirdparty libraries are changed, we only need to change one place